practices/communication.md

```diff
@@ -0,0 +1,62 @@
+You can set your business time using Google Calendar. With that, everybody will see it when scheduling a meeting with you.
+Add this information to your Slack profile. It will be available with a single click on your Slack picture. We recommend this format `<job title> @ <squad/team/department> - Working hours: X AM - Y PM (<timezone>)`
+
+### No Hello
+
+When sending a message, like in Slack, we usually try to be polite by not jumping right into the request. This is not a good practice and costs time. The [No Hello](https://nohello.net) website presents a better way to do it. Please take a look.
+<div align="center">
+  <img alt="No Hello" src="./no-hello.png" width="600" />
+  <br/>
+  <i>Image by <a href="https://nohello.net">No Hello</a></i>
+</div>
+
+### Slack Notifications
+
+It is common and useful to have Slack installed on our phones. With this, we got the notifications as well. They are very useful when we are working, but they can be very unhealthy. Receiving notifications when you are out of your business time will prevent you from disconnecting and resting. Within some time, it will cause a lot of stress. That's why we recommend setting Slack to send mobile notifications during your business hours or for really urgent channels only.
+
+
+### Slack Status
+
+Slack is your front door. People will look for you there when they need to ask or tell something. It's important to make clear if you are available or not. For that, we can use the Slack Status. For example: Out of Office, In a Meeting, Internet Connection Unstable. Setting the integration between Google Calendar and Slack will update your status automatically.
+
+### Slack Threads
+
+Threads can be used to organize discussions in Slack. Multiple subjects can flood channels with messages. With threads, reply messages can be tied to their original message keeping readers notified with every new message and without distracting others in the channel that might not have an interest in that particular subject. 
+
+### Collaborative Docs
+
+We have many meetings to discuss ideas or proposals. Instead of scheduling a meeting whenever a new project or problem arises, try writing about it in a Google Doc, so the team can use the comments features to have an async discussion. This way, we give people time to understand and evaluate the subject, and they can make a decision without trying to fit everyone in a call at the same time. This feature is also available in other tools like Notion.
+
+
+### How to Solve a Fire
+
+It takes time to figure out how to solve a fire, and there is always a lesson learning. This learning should be shared so that we save time in the future. After solving a fire, we recommend writing a how to fix it. The next firefighters will thank you for that.
```

Comment:
```suggestion
There are fires of different sizes. Yet there's always lessons to be learned. This learning should always be shared with the team so that we save time in the future. Whenever you solve one of those difficult fires, try to cover that case with a regression test. We also recommend doing a write up on how to fix it, if applicable. The next firefighters will thank you for that.
```